### PR TITLE
Peer Review Submission

### PR DESCRIPTION
**Pull request by jack-maber**

### CODE REVIEW

File: Main.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+def bluebell(window):
+    for y in xrange(HEIGHT):
+        for x in xrange(WIDTH):
+            Red = window.get_at((x, y)).r
+            Green = window.get_at((x, y)).g
+            Blue = window.get_at((x, y)).b
+            pxarray[x, y] = (Red, Green, 255 - Blue)
+
+
```
Comment (Herro-Sama) [inline Main.py:82]: Couldn't you have created a class that holds you Y and X ranges?

File: Main.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            redblack()
+        if event.type == KEYDOWN and event.key == K_j:
+            invert(window)
+        if event.type == KEYDOWN and event.key == K_k:
+            nightvision(window)
+        if event.type == KEYDOWN and event.key == K_f:
+            ganggreen(window)
+        if event.type == KEYDOWN and event.key == K_d:
+            bluebell(window)
+
```
Comment (Herro-Sama) [inline Main.py:104]: You can create a short hand for event.type which would have made your if statements much shorter

**Review by Herro-Sama — COMMENTED** [review]
Seems solid to me, you could have somethings in classes and you could have created short hand for event.type but otherwise I think it's solid

File: Main.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+window = pygame.display.set_mode((WIDTH, HEIGHT)) #Creates window
+pygame.display.set_caption('Meme Generator 3000') #Sets caption for game window
+window.blit(imagelistRan,(0, 0)) #Blits image to created window
+
+textlist = ["CJ?","Are you OK?","Excuse me Young Man!","WHEN YOU MEME"]
+textlist1 = ["Ohhhhhhhhhh","Big Smoke, It's me!","Boiiiiiiiiiiiiiiiiiiiiiiiii","It was me, Dio!","JAGUARS","BOTTOM TEXT"]#Picks random text snippet from list
+textlistRan = (random.choice(textlist))
+textlistRan1 =(random.choice(textlist1))
+
+def greyscale():
```
Comment (KieranGolds) [inline Main.py:28]: Clean classes but not a lot of comments from here on

File: Main.py (hunk 2 above)
Comment (RobertsOz) [reply Main.py:104]: Why not just do 1-6 for the keys, instead of g,h,j,k,f,d

File: Main.py [hunk 4]
```diff
@@ -0,0 +1,117 @@
+import pygame, sys, time, math, random
```
Comment (hannahlu) [inline Main.py:1]: Could write each import statement on a separate line, according to PEP8.

File: Main.py [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+pygame.init()  #  Initiates Pygame
+img1 = pygame.image.load('BigSmoke.jpg')
+img2 = pygame.image.load('Harambe.jpg')
+img3 = pygame.image.load('Doge.jpg')
+img4 = pygame.image.load('YeaBoi.jpg')
+img5 = pygame.image.load('Dio.jpg')
+img6 = pygame.image.load('Jaguars.jpg')
+img7 = pygame.image.load('Issac.jpg')
+myfont = pygame.font.SysFont("comic", 100) #Imports text font
```
Comment (hannahlu) [inline Main.py:12]: Try writing comments to explain blocks of code rather than lots of inline explanations.

**Pull request closed without merging**